src/request.rs

```diff
@@ -49,7 +49,14 @@ pub enum Request {
     GetUnpubIData(XorName),
     PutUnpubIData(UnpubImmutableData),
     DeleteUnpubIData(XorName),
-
+    DeleteMData {
+        // Address of the mutable data to be fetched
+        address: MutableDataRef,
```

Comment:
I think we might have to add the specification of `MutableDataKind` (as in `unsequenced` or `sequenced`) to disambiguate it further, but since both types exist in the same name space, I can't see any immediate disadvantages to having the same RPC for both without having to specify the actual type. So I'll leave it as a comment and we can discuss this later.